scripts/dev/add_to_project.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# call with the PR number as the only command line argument
+
+# get the PR num from the command line argument
+PR_NUM=$1
+
+# the GraphQL Project ID can be retrieved from a given organization's project, where the URL is:
+#  https://github.com/orgs/ORGANIZATION/projects/SOME_PROJECT_NUMBER/views/2
+# and the associated call to graphql is:
+#gh api graphql -f query='
+#  query{
+#    organization(login: "ORGANIZATION"){
+#      projectV2(number: SOME_PROJECT_NUMBER) {
+#        id
+#      }
+#    }
+#  }'
+# TODO: Just specify the project organization and number and get the graphql ID in here
+PROJ_ID=PVT_kwDOAB0YcM4AEWD7
```

Comment:
This looks like a secret token, but it's not.  It's just an ID for the GitHub project we are adding the PR to, and having this ID does not grant any access to anything.  That said, as I comment here, I would like to just query this from meaningful keywords instead and remove this hardcoded ID just to make it easier to adapt for other projects later.